model/src/trade.rs

```diff
@@ -0,0 +1,62 @@
+//! Contains the Trade type as described by the specification with serialization as described by the openapi documentation.
+
+use num_bigint::BigUint;
+use primitive_types::{H160, U256};
+use serde::{Deserialize, Serialize};
+// use serde_with::serde_as;
+use crate::order::OrderUid;
+
+#[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, Hash)]
+#[serde(rename_all = "camelCase")]
+pub struct DbTrade {
+    pub order_uid: OrderUid,
```

Comment:
I don't think this struct should live here because it is only relevant for the db module where it used to be defined.

Reply:
Moved it back!